fugue/workflow/workflow.py

```diff
@@ -1174,8 +1175,8 @@ def add(self, task: FugueTask, *args: Any, **kwargs: Any) -> WorkflowDataFrame:
         for v in dep.dependency.values():
             v = v.split(".")[0]
             self._graph.add(name, v)
-            if len(self._graph.down[v]) > 1 and self.conf.get(
-                "fugue.workflow.auto_persist", False
+            if len(self._graph.down[v]) > 1 and self.conf.get_or_throw(
+                FUGUE_CONF_WORKFLOW_AUTO_PERSIST, bool
             ):
                 self._spec.tasks[v]._persist = self.conf.get(
```

Comment:
Any reason why you didn't change the one below this line to a constant also?

Reply:
Great you see it, I am working on it right now. It requires more refactoring.
This code review is half done